cluster/addons/metadata-proxy/gce/metadata-proxy.yaml

```diff
@@ -40,13 +42,22 @@ spec:
           limits:
             memory: "32Mi"
             cpu: "50m"
+      # BEGIN_PROMETHEUS_TO_SD
+      - name: prometheus-to-sd-exporter
+        image: gcr.io/google_containers/prometheus-to-sd:v0.1.2
```

Comment:
@loburm Should this be v0.2.2?

Reply:
Yes, please use 0.2.2 (as I remember 0.1.2 even don't have --source flag). Also it may have sense to add pod_name and pod_namespace, see https://github.com/kubernetes/kubernetes/pull/54635

Just pay attention that in future we will change pod name to pod id (currently not all kubernetes downward api support it)